app/serializers/code_school_serializer.rb

```diff
@@ -1,8 +1,4 @@
 class CodeSchoolSerializer < ActiveModel::Serializer
   attributes :name, :url, :logo, :full_time, :hardware_included, :has_online, :online_only, :mooc, :is_partner, :rep_name, :rep_email
   has_many :locations
-
-  def locations
-    object.locations.order(:state).order(:city)
-  end
```

Comment:
With the sort order for locations added a default to the CodeSchool model's locations association, this custom method is no longer necessary. This was part of the N+1 query.